WooCommerce/Classes/POS/Utils/POSFontStyle.swift

```diff
@@ -5,35 +5,32 @@ import SwiftUI
 enum POSFontStyle {
```

Comment:
Figma also has `underline` style. We could of course do it separately/later. The approach may need to be a bit different, although there looks to be[ a modifier that can be applied](https://developer.apple.com/documentation/swiftui/text/underline(_:color:)).

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/15114/commits/6378adf7f4acbf0d95405866f95d88df43cd2e20. With the associated value `underline: Bool` in the regular font cases, all regular font use cases require `*Regular()` now but I think the code is more maintainable now.